### DISCUSSION TITLE
Tag discord roles on/during event creation

### DISCUSSION THREAD

**Discussion by Dovakillz** [gameserverapp/Platform#920] [Suggestions]
When creating an event on the website/calendar it woulf be cool to have the option to tag specific roles in the discord upon completion of the event setup so when it posts into the discord channel it automatically tags the role/s you set it to tag.

**Comment by WEchoW** [comment]
Unless you create discord roles with same name(as I did) same you have on the website I don't think there is a possibility to "tag" them

**Comment by mvaessen** [comment]
Thanks for the suggestion, this will available on the next update
